HostWormsGame.ps1

```diff
@@ -0,0 +1,38 @@
+$yourSecretSlackToken = 'ENTER YOUR SECRET SLACK TOKEN' # Get from https://api.slack.com/web#authentication
+
+$channel = '#games-worms'
+$optionsRepoUrl = 'https://api.github.com/repos/TheEadie/WormsLeague/releases/latest'
+$installDirPath = 'C:\Program Files (x86)\Steam\steamapps\common\Worms Armageddon\'
+$schemesDirPath = Join-Path $installDirPath '\User\Schemes\'
+
+#Gets the ip that'd actually be used rather than random virtual netadapters.
+function Get-Ip() {
+	$pingResponse = (ping -4 -n 1 $env:computername)[1]
+	$ipStart = $pingResponse.IndexOf("[") + 1 # Hopefully the machine name doesn't contain "[" or "]"
+	$ipLength = $pingResponse.IndexOf("]") - $ipStart
+	return $pingResponse.SubString($ipStart, $ipLength)
+}
+
+function Send-Slack() {
+	$ip = Get-Ip
+	$messageText = "@here Hosting at: $ip"
```

Comment:
More details here: https://api.slack.com/docs/formatting#linking_to_channels_and_users

(looks like you could also set `parse=full` as one of the api request params)